### ISSUE TITLE
Show members won through the campaign

### ISSUE THREAD

**Issue by bkaless** [systopia/de.systopia.campaign#15]
The extension should list the members which has filled the appropriate campaign-field.

**Comment by bjendres** [comment]
Unfortunately, the contact entity doesn't have a campaign field. What you would have to do is connect them e.g. via a (custom) "New Contact Aquired" activity (those do have a campaign field), and then evaluate that by implementing the KPI hook. There's probably other sensible approaches to this, too, but I don't see any standardised structure that can be included into the extension as a standard KPI.

**Comment by bkaless** [comment]
But the membership has a campaign field, hasn't it?

**Comment by bjendres** [comment]
Yes, that's true - sorry. We could add a standard KPI for that...
